src/Sulu/Bundle/MediaBundle/Api/Media.php

```diff
@@ -668,7 +682,7 @@ public function getChanger()
      */
     public function getCreated()
     {
-        return $this->getFileVersion()->getCreated();
```

Comment:
The created date for a specific fileVersions are under 'versions' http://sulu.lo/admin/api/media/1